packages/beacon-node/src/execution/engine/disabled.ts

```diff
@@ -30,4 +30,8 @@ export class ExecutionEngineDisabled implements IExecutionEngine {
   getState(): ExecutionEngineState {
     throw Error("Execution engine disabled");
   }
+
+  getClientVersion(): Promise<ClientVersion[]> {
```

Comment:
why is this an array of client versions?

Reply:
> why is this an array of client versions?

I believe the spec mandates to return multiple client versions because CL may be connected to a multiplexer. So multiplexer will need to return multiple client versions whereas EL will return an array of single client version if connected directly.